app/services/jobseekers/job_applications/prefill_job_application_from_previous_application.rb

```diff
@@ -78,13 +78,10 @@ def completed_steps
   end
 
   def in_progress_steps
-    if previous_application_was_submitted_before_we_began_validating_gaps_in_work_history?
-      %w[employment_history]
-    elsif !previous_application_has_professional_status_details?
-      %w[professional_status]
```

Comment:
Oh so these two were "exclusive". It shouldn't have been like that for starters, right? It is not a one or another situation but both steps could be needed on the same Job application.